ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -303,11 +304,14 @@ forkBlockForging maxTxCapacityOverride IS{..} blockForging =
             mkCurrentBlockContext currentSlot
                 <$> ChainDB.getCurrentChain chainDB
           case eBlkCtx of
-            Right blkCtx -> return blkCtx
+            Right blkCtx -> do
+              return blkCtx
```

Comment:
Is the idea here to improve visual consistency?

Reply:
Oh, that's an accident, I first added the trace message in this case. I'll revert.